### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by phavekes**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+    "@symfony/webpack-encore": "^1",
     "@types/jest": "^24",
     "@types/jquery": "^3.3.9",
     "@types/node": "^10",
     "compass-mixins": "^0.12.10",
     "css-loader": "^5",
     "import-glob-loader": "^1.1.0",
-    "jest": "^24",
-    "node-sass": "^4.5.3",
+    "jest": "^27",
```
Comment (MKodde) [inline package.json:10]: :crossed_fingers:

File: package.json [hunk 2]
```diff
@@ -26,6 +26,8 @@
     "yargs-parser": ">=18.1.2"
   },
   "resolutions": {
-    "yargs-parser": ">=18.1.2"
+    "yargs-parser": ">=18.1.2",
```
Comment (MKodde) [inline package.json:29]: Consider removing the old resolution. Chances are, it is no longer required. And having less resolutions is a win in my book!

**Review by MKodde — APPROVED** [review]
Good to go, consider trying the removal of the yargs-parser resolution, but feel free to move ahead without doing so. 

Thanks for picking up this chore! The Yarn-Fu is strong with @phavekes

**Comment by phavekes** [comment]
@MKodde should we attempt to update jest and repair the tests, or can we leave it this way, since it's a dev-dep only?

**Comment by MKodde** [comment]
I'm happy leaving Jest on an older version. Chances are we'll need to upgrade it in the near future, but fine for now!

**Pull request merged**